docs/howtos/verification-config.md

```diff
@@ -1,10 +1,11 @@
 ---
 sidebar_label: Verification configuration
+sidebar_position: 45
 title: Verification configuration
 description: Verification configuration for Kubewarden.
 keywords: [kubewarden, kubernetes, verification configuration]
 doc-persona: [kubewarden-operator, kubewarden-integrator]
-doc-type: [tutorial, howto, reference]
+doc-type: [howto]
```

Comment:
I think this Verification configuration belongs under "reference". For this "verification configuration", I would:
- Swap the order of the contents in the file, first putting the "format", then the "example".
- Add a starting phrase stating what is it for: 
  > this `verification-config` format is used by `policy-server` to verify policy modules provenance, and
  > by `verify-image-signatures` policy to verify cluster images provenance. See ./howtos/secure-supply-chain for more info.
- Move the subsection of https://deploy-preview-353--silly-bunny-8cedd0.netlify.app/next/howtos/secure-supply-chain#signature-configuration-reference, which is a reference, inside the "Verification Configuration" page.
- Potentially rename from "Verification configuration" to "verification-config format".

Reply:
OK, that's in the next commit. Thanks.